tests/playbooks/plugin/touch-vmi.yaml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Touch a file on all the VMIs from the KubeVirt inventory plugin
+  hosts: all
+  vars:
+    ansible_user: fedora
+    ansible_ssh_pass: fedora
+  tasks:
+    - name: Ansible create file if it doesn't exist example
```

Comment:
How are we going to verify whether it was done?

Reply:
That the task didn't fail, I think that's good enough.

Reply:
One thing is to check whether task did not fail and the other whether it did what was expected. Maybe we could enable and start httpd or something similar. It would be easier to verify whether the task succeeded.

Reply:
Sure we can, or we can use `stat` module and check that the file is created.

Reply:
Added